
version: "3.9"
services:
  anaconda3:
    container_name: anaconda
    image: continuumio/anaconda3:2021.11
    ports:
      - "8888:8888"
    volumes:
    - ${HOME}/Projects/work/datasets:/datasets
    - ${HOME}/Projects/work/notebooks:/notebooks
    - ${HOME}/Projects/work/repo:/repo
    networks:
      - anaconda    
    command: bash -c " \
        conda install jupyter -y --quiet && \
        mkdir -p /notebooks && \ 
        jupyter notebook \
        --notebook-dir=/notebooks --ip='*' --port=8888 \
        --no-browser --allow-root "      

networks:
  anaconda:
    external: true
